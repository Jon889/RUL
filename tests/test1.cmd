./RUL hsr2.txt print 0x0d001a00
