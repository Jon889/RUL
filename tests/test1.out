  +-------+  +-------+   +-------+
  |    0  |  |    0  |   |    0  |
M:|0     1|O:|0     1|N2:|0     0|	0x0d001a00,0,0 (null)
  |    3  |  |    3  |   |    0  |
  +-------+  +-------+   +-------+

  +-------+  +-------+   +-------+
  |    0  |  |    0  |   |    0  |
M:|3     0|O:|3     0|N2:|0     0|	0x0d001a00,1,0 (null)
  |    1  |  |    1  |   |    0  |
  +-------+  +-------+   +-------+

  +-------+  +-------+   +-------+
  |    3  |  |    3  |   |    0  |
M:|1     0|O:|1     0|N2:|0     0|	0x0d001a00,2,0 (null)
  |    0  |  |    0  |   |    0  |
  +-------+  +-------+   +-------+

  +-------+  +-------+   +-------+
  |    1  |  |    1  |   |    0  |
M:|0     3|O:|0     3|N2:|0     0|	0x0d001a00,3,0 (null)
  |    0  |  |    0  |   |    0  |
  +-------+  +-------+   +-------+

  +-------+  +-------+   +-------+
  |    0  |  |    0  |   |    0  |
M:|3     0|O:|3     0|N2:|0     0|	0x0d001a00,0,1 (null)
  |    1  |  |    1  |   |    0  |
  +-------+  +-------+   +-------+

  +-------+  +-------+   +-------+
  |    0  |  |    0  |   |    0  |
M:|0     1|O:|0     1|N2:|0     0|	0x0d001a00,1,1 (null)
  |    3  |  |    3  |   |    0  |
  +-------+  +-------+   +-------+

  +-------+  +-------+   +-------+
  |    1  |  |    1  |   |    0  |
M:|0     3|O:|0     3|N2:|0     0|	0x0d001a00,2,1 (null)
  |    0  |  |    0  |   |    0  |
  +-------+  +-------+   +-------+

  +-------+  +-------+   +-------+
  |    3  |  |    3  |   |    0  |
M:|1     0|O:|1     0|N2:|0     0|	0x0d001a00,3,1 (null)
  |    0  |  |    0  |   |    0  |
  +-------+  +-------+   +-------+

